libs/schema-profunctor/src/Data/Schema.hs

```diff
@@ -907,6 +907,12 @@ instance ToSchema Bool where schema = genericToSchema
 
 instance ToSchema Natural where schema = genericToSchema
 
+instance (ToSchema a) => ToSchema (Identity a) where schema = dimap runIdentity Identity schema
+
+instance (ToSchema a) => ToSchema (Const a x) where schema = dimap getConst Const schema
+
+instance (S.ToSchema a, A.ToJSON a, A.FromJSON a) => ToSchema (Maybe a) where schema = genericToSchema
+
```

Comment:
I am not sure how that behaves.
If we want to be consistent with the rest of our code base, we can just define the part in the ToSchema instance as follows:
```
<*> (.membersCount) .= maybe_ (optField "membersCount" schema)
```